src/v0/util/facebookUtils/index.js

```diff
@@ -292,7 +292,13 @@ const formingFinalResponse = (
   throw new TransformationError('Payload could not be constructed');
 };
 
+const isHtmlFormat = (string) => {
+  const pattern = /<([a-z]+)\b[^>]*>(.*?)<\/\1>/i;
```

Comment:
This regex would even validate if tomorrow the integration sends xml response right ?!

Reply:
regex should be able to identify XML format also

Reply:
it('should return true for xml doctype', () => {
    expect(
      isHtmlFormat(
        '"<note><to>Tove</to><from>Jani</from><heading>Reminder</heading><body>Don\'t forget me this weekend!</body></note>"',
      ),
    ).toBe(true);
  });
  
  this case passed

Reply:
If that is the case, then this function should be renamed to something else. As others might use it thinking it is for HTML only but this function identifies XML as well.
(Or)
Update the regex to identify only HTML & discard any other kind of output(XML)

Reply:
done to identify html only